ios/RNCImageEditor.xcodeproj/project.pbxproj

```diff
@@ -7,9 +7,9 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
```

Comment:
I am 99% sure we can remove that whole .pbxproj file

Reply:
i will remove it